PendingReleaseNotes.md

```diff
@@ -16,6 +18,11 @@ an example usage
 - Git tags can be added for alpha, beta, or rc releases. For example: v1.1.0-alpha.0
 - Added `deviceClass` to the per OSD device specific `config` section for setting custom crush device class per OSD.
 - Use `--db-devices` with Ceph 14.2.1 and newer clusters to explicitly set `metadataDevice` per OSD.
+- Rook [EdgeFS](http://edgefs.io) Operator CRDs now graduated to stable v1.
```

Comment:
Could you move these notable features to the EdgeFS section? This section is intended to be for general project features, then a section below alphabetically for each provider. It looks like we've had a few other storage-provider specific items sneak into this section as well. While you're at it, could you move these?
- Ceph: Lines 8, 13, 16, 19, 20
- YugabyteDB: Line 17
Since each section is so long, we could use links at the top to each storage provider, but we can do that when the final release notes are created if not now.

Reply:
Ok. I renamed section to "Notable Rook Framework Features", so that there is no confusion. Also, I thought it make sense to keep YugabyteDB because it is our new provider that framework now supports.